src/controllers/class.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express');
 
-const Class = require('../models/Class');
+const Class = require('../models/class');
```

Comment:
For my understanding the team wanted the models to be with capital first letter. is this change intended?

Reply:
I know but the server was unable to run with those capitalized so I had to change them back in order to push

Reply:
I'm guessing classes and subscriptions will change their respective pathings in their forthcoming PRs